<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="Floormap">
        <item>
         <layout class="QHBoxLayout" name="FloormapHeader">
          <item>
           <widget class="QPushButton" name="previousFloormapButton">
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="FloomapsComboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextFloormapButton">
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="FloormapScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="FloormapScrollAreaHolder">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>833</width>
             <height>572</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QGridLayout" name="FloormapScrollAreaContents"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="Waitlist">
        <item>
         <layout class="QHBoxLayout" name="WaitlistHeader">
          <item>
           <widget class="QPushButton" name="addToWaitlistButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="WaitlistScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="WaitlistScrollAreaHolder">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>279</width>
             <height>573</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="WaitlistScrollAreaContents"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDelete">
    <property name="title">
     <string>Delete</string>
    </property>
    <addaction name="actionDelete_All_Parties_On_Waitlist"/>
    <addaction name="actionDelete_All_Tables"/>
   </widget>
   <addaction name="menuDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDelete_All_Parties_On_Waitlist">
   <property name="text">
    <string>Delete All Parties On Waitlist</string>
   </property>
  </action>
  <action name="actionDelete_All_Tables">
   <property name="text">
    <string>Delete All Tables</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
